src/Controls/src/Core/Handlers/Items/Android/MauiRecyclerView.cs

```diff
@@ -540,6 +540,13 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 			_scrollHelper?.AdjustScroll();
 		}
 
+		public override bool OnTouchEvent(MotionEvent e)
+		{
+			if (!Enabled)
```

Comment:
I think at this point should also take into account the `InputTransparent` property.

Reply:
@jsuarezruiz,
Without my changes, InputTransparent is already working fine for CollectionView. Let me know if any additional modifications are needed.